software/database/pom.xml

```diff
@@ -144,11 +144,24 @@
         </dependency>
 
         <dependency>
+            <groupId>${jclouds.groupId}.provider</groupId>
```

Comment:
We already depend on `brooklyn-locations-jclouds`, which should bring in this dependency. Why did it need to be added?